Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/Joystick/JoyStickView.swift

```diff
@@ -163,14 +146,14 @@ class JoyStickView: UIView {
     }
 
     func startUpdateTimer() {
-        if mUpdateTimer == nil {
+        if let _ = mUpdateTimer {
```

Comment:
This modification is not correct, it should check if the `mUpdateTimer` is nil, otherwise, the joystick button cannot move around.